device/device.yaml

```diff
@@ -17,21 +17,35 @@ components:
       x-include: 
       - '../common/common.yaml#/components/schemas/Named.Object'
       description: >-
-        A container for emulated interfaces and protocol devices.
+        A container for emulated interfaces, loopback interfaces and protocol configurations.
       type: object
-      required: [container_name, ethernet]
       properties:
-        container_name: 
+        ethernets:
           description: >-
-            The unique name of a Port or Lag object that will contain the 
-            emulated interfaces and/or protocol devices.
-          type: string
-          x-constraint:
-          - '/components/schemas/Port/properties/name'
-          - '/components/schemas/Lag/properties/name'
-        ethernet:
+            The emulated Ethernet settings.
```

Comment:
Ethernet configuration for one or more emulated network interfaces.

Reply:
ok.